tests/test_api.py

```diff
@@ -0,0 +1,77 @@
+import unittest
+import pymacaroons
+import responses
+from webapp.app import app, _api_request, ANBOXCLOUD_API_BASE
+
+
+class TestApi(unittest.TestCase):
+    def setUp(self):
+        """
+        Set up Flask app for testing
+        """
+        app.testing = True
+        self.client = app.test_client()
+
+    @responses.activate
+    def test_api_request(self):
+        """
+        Simple mock tests of the _api_request handler
```

Comment:
I don't think we should be testing the `_api_request` handler (which is a private, internal function). If this function isn't working properly, it should be exposed by testing any one of the *public* API functions which rely on it.

We can achieve 100% coverage by purely testing the public interface, and then we would give the module complete flexibility to change the internal logic (e.g. rename `_api_request`) without having to change any tests.

I would simply remove this test.